src/test/kotlin/com/r3/corda/finance/obligation/SWIFTObligationTestsWithOracleAutomatic.kt

```diff
@@ -0,0 +1,28 @@
+package com.r3.corda.finance.obligation
```

Comment:
I think we can remove the old test SWIFTObligationTestsWithOracle in this PR. The old API compatibility shouldn't be required once the new one works.